ui/apps/platform/src/Containers/MainPage/Navigation/utils.ts

```diff
@@ -9,7 +9,7 @@ import { HasReadAccess } from 'hooks/usePermissions';
 // Parent example: Vulnerability Management (1.0) if Vulnerability Management (2.0) is rendered and so on.
 type TitleCallback = (navDescriptionFiltered: NavDescription[]) => string | ReactElement;
 
-type IsActiveCallback = (pathname: string) => boolean;
+type IsActiveCallback = (location: Location) => boolean;
```

Comment:
Instead of extracting the `pathname` from the `location` here, we expect the full `location` object so that we can handle link "active" states based on both the `pathname` and `search` fields.